lib/Conversion/ATenToXTenPass.cpp

```diff
@@ -191,6 +192,16 @@ bool isAdaptiveAvgPoolGlobalAveragePool2D(Value outsizes) {
   return false;
 }
 
+bool checkLinearForXten(Value input, Value weights, Value bias) {
+  // Ensure that when the bias is given, it is of rank 1
```

Comment:
Why is the bias always of rank 1 here?

Reply:
torch defines their bias in linear to be of rank 1. I just kept the check. They assume you can broadcast the tensor to 2D.